### ISSUE TITLE
apk add using variables fails

### ISSUE THREAD

**Issue by antonmarin** [hadolint/hadolint#252]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
like this
```
▶ cat Dockerfile
FROM php:7.2-fpm-alpine
RUN apk add --no-cache autoconf=2.69-r0 g++=6.4.0-r5 (Actually i'v got ${VARIABLE} here instead packages)

PhpstormProjects/tests/docker-experiments                                                                                                  
▶ docker run --rm -v ${PWD}:/app hadolint/hadolint:v1.8.0 hadolint /app/Dockerfile
```
### Actual behavior
```
▶ cat Dockerfile
FROM php:7.2-fpm-alpine
ENV VARIABLE autoconf=2.69-r0 g++=6.4.0-r5
RUN apk apk add --no-cache ${VARIABLE}
                                                                                               
▶ docker run --rm -v ${PWD}:/app hadolint/hadolint:v1.8.0 hadolint /app/Dockerfile
/app/Dockerfile:4 SC2086 Double quote to prevent globbing and word splitting.
/app/Dockerfile:4 DL3018 Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`
```
### Steps to reproduce the behavior
1. Create Dockerfile with content
```
FROM php:7.2-fpm-alpine
ENV VARIABLE autoconf=2.69-r0 g++=6.4.0-r5
RUN apk apk add --no-cache ${VARIABLE}
```

2. `docker run --rm -v ${PWD}:/app hadolint/hadolint:v1.8.0 hadolint /app/Dockerfile`

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter v1.10.4-3-g20c5c3b
```

**Dockerfile** (if relevant)

```Dockerfile
FROM php:7.2-fpm-alpine
ENV VARIABLE autoconf=2.69-r0 g++=6.4.0-r5
RUN apk apk add --no-cache ${VARIABLE}
```

**Additional environment details (OS, stack version, etc.)**
MacOS 10.13.6

**Comment by lorenzo** [comment]
currently hadolint does no variable substitutions at all, for the time being I suggest that either you don’t use a variable, or add a
comment above for ignoring the rule.

**Comment by zemanlx** [comment]
or use it per package like in node [Dockerfile](https://github.com/nodejs/docker-node/blob/e3ec2111af089e31321e76641697e154b3b6a6c3/8/alpine/Dockerfile)

```Dockerfile
FROM php:7.2-fpm-alpine

ENV AUTOCONF_VERSION=2.69-r0 \
    GPP_VERSION=6.4.0-r5

RUN apk add --no-cache \
    autoconf=${AUTOCONF_VERSION} \
    g++=${GPP_VERSION}
```

I find this style more useful. Also if you do not need it after build using `ARG` seems reasonable.

**Comment by antonmarin** [comment]
I use variable $PHPIZE_DEPS from php image, so can't change it's content. And ignoring isn't a right way too, you know.

Are you going to implement variable substitution?

**Comment by antonmarin** [comment]
thanks for help
